server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -34,7 +34,10 @@
   class="usa-card tablet-lg:grid-col-6 widescreen:grid-col-4 cf-application-card"
   th:classappend="${showImage} ? usa-card--header-first : '' "
 >
-  <div class="usa-card__container">
+  <div
+    class="usa-card__container"
+    th:classappend="${card.isCommonIntakeForm} ? bg-primary-lighter : '' "
```

Comment:
**suggestion (non-blocking):** We might want to also add the condition that the card `sectionType` is 'MY_APPLICATIONS'.  Then, it'll be done for Anna when she works on the un-submitted intake form.

Reply:
i think the unsubmitted card is the same color, actually. It just spans the full width of the page.